include/eredis.hrl

```diff
@@ -6,6 +6,7 @@
 -type option() :: {host, string() | {local, string()}} |
                   {port, inet:port_number()} |
                   {database, integer()} |
+                  {username, string()} |
```

Comment:
```erl
string() | fun(() -> string())
```

Reply:
changed to `iodata() | fun(() -> iodata()) | undefined`